src/server/components/lists/find/handlers/__tests__/results/get.test.ts

```diff
@@ -0,0 +1,47 @@
+import * as lawyers from "./../../../../searches/lawyers";
+import { get } from "./../../result";
+import { HttpException } from "../../../../../../middlewares/error-handlers";
```

Comment:
Would it be easier to use path aliases here?

Reply:
Yeah VSCode struggles with the path aliases for test files as well. Not sure if there's a setting that needs to be changed somewhere 🤔

Reply:
I think it's okay to keep it if it's better for your IDE. Not sure if it's worth having a spike ticket to investigate this 🤔